app/src/main/java/com/example/sharewhatyoucanproject/user/UserFragment.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.sharewhatyoucanproject.user
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.sharewhatyoucanproject.R
+import com.example.sharewhatyoucanproject.databinding.FragmentUserBinding
+
+class UserFragment : Fragment() {
+
+    private var _binding: FragmentUserBinding? = null
+    private val binding get() = _binding!!
```

Comment:
make sure to null out the binding in `onDestroyView`

Reply:
done